test/bundler/bundler_splitting.test.ts

```
@@ -1777,6 +1778,432 @@ describe("bundler", () => {
+      "/other.js": `export const nav = () => import("./r2.js")`,
+      ...preloadChainFiles(2),
+    },
+    entryPoints: ["/entry.js", "/other.js"],
+    splitting: true,
+    outdir: "/out",
+    target: "browser",
+    publicPath: "https://cdn.example.com/v1/",
+    onAfterBundle(api) {
+      const entry = api.readFile("/out/entry.js");
+      expect(entry).toMatch(/__preload\("(\w+)"\), import\("https:\/\/cdn\.example\.com\/v1\/r0-\1\.js"\)/);
+      const [ids, nodes] = eval("[" + entry.match(/__chunks\(import\.meta\.url,(\[.*\]),0\);/)![1] + "]") as [
```

Biome (2.5.8)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)